recipes/gromacs/2020/meta.yaml

```diff
@@ -1,16 +1,20 @@
 # Update from dwnload page, e.g. http://manual.gromacs.org/2019.1/download.html
 # For latest release, see
 # http://manual.gromacs.org/documentation/
-{% set name = "gromacs" %}
 {% set version = "2020.4" %}
 {% set md5sum = "aead694ec7d1222cbc40011c88047c3d" %}
+{% set build = 1 %}
+{% set build = build + 100 %}  # [mpi == 'nompi']
 
 package:
-  name: {{ name }}
+  name: gromacs
   version: {{ version }}
 
 build:
-  number: 0
+  number: {{ build }}
+  string: >-
+    nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi == 'nompi']
+    mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi != 'nompi']
```

Comment:
The `nompi_`/`mpi_mpich_`/`mpi_openmpi_` build string prefixes are commonly used at conda-forge to allow users to explicitly request one of those variants with, e.g., `gromacs=*=mpi_openmpi_*`.

Reply:
ref http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=openmpi#including-a-no-mpi-build